requirements/edx/github.in

```diff
@@ -57,6 +57,7 @@
 -e git+https://github.com/edx/django-wiki.git@0.0.27#egg=django-wiki
 git+https://github.com/edx/MongoDBProxy.git@d92bafe9888d2940f647a7b2b2383b29c752f35a#egg=MongoDBProxy==0.1.0+edx.2
 -e git+https://github.com/jazkarta/edx-jsme.git@690dbf75441fa91c7c4899df0b83d77f7deb5458#egg=edx-jsme
+git+https://github.com/edx/django-ratelimit-backend.git@v2.0.1a1#egg=django-ratelimit-backend==2.0.1
```

Comment:
@cpennington some context would be helpful.  Is this a premanent fork?  There's no PR back upstream to get rid of the fork and no reference to a ticket for follow up to clean this up.  Can you provide what the plan is for that?

Reply:
clarification: add the ticket and PR links in a comment above this so we know how we're going to get off this for, and then I think this is good to go.

Reply:
Done.